timescale-forge/vpc-peering.md

```diff
@@ -0,0 +1,141 @@
+# VPC Peering in Timescale Forge
+
+Timescale Forge allows you to create a private network peering connection between
+your cloud VPC and the Timescale Forge cloud infrastructure where your Timescale
+services are hosted.
+
+VPC peering isolates your Timescale Forge services ensuring that they are only
+accessible via your peered VPC, providing a greater security model due to reduced
+attack vector.
+
+The Timescale Forge console makes creating and configuring your VPC peering connections
+simple, providing controls for adding and removing VPC peering connections, migrating
+services to and from VPCs, and creating new services with VPC peering attachments.
+
+## Requirements and limitations
+To use Timescale Forge VPC peering, you need your own cloud VPC, where your
+applications and infrastructure are already running.
+
+If you do not have administrative access to your cloud provider account, you will need
+to work with someone from your team with sufficient permissions to:
+
+- accept VPC peering requests,
+- configure route table rules,
+- configure security group and firewall rules.
+
+<highlight type="tip">
+The Timescale Forge VPC peering feature is currently limited to AWS VPC peering only.
+</highlight>
+
+<highlight type="tip">
+Timescale Forge defaults to a limit of 3 VPCs per Forge project. If more VPCs are needed,
+you may contact support to request a quota increase.
+</highlight>
+
+<highlight type="warning">
+Once you have attached your Timescale Forge service to a VPC, it will no longer be accessible
+via the public internet. It will only be accessible via your AWS VPC which has been peered
+with your Timescale Forge VPC.
+
+Timescale Forge uses a different DNS name for a Timescale service once it has been attached
+to a VPC. This means that you will need to update your connection string if migrating a service
+from the public internet into a VPC, or vice-versa.
+</highlight>
+
+## Setup
+Before you begin, log in to the Timescale Forge console.
+
+### Create a new VPC
+In the Timescale Forge console, click `VPC` in the left navigation bar to go to the VPC
+dashboard. You can add new VPCs here for your Timescale Forge services to attach to.
+The VPCs created here are peered with your own VPC as part of the setup process.
+
+<img class="main-content__illustration" src="vpc-dashboard.png" alt="Navigate to the VPC dashboard in the console"/>
+
+Click `Create VPC`, type a name for your new VPC, and provide an IPv4 CIDR block.
+Make sure that the CIDR block you choose for your VPC does not overlap with the
+cloud VPC you are using to create a peering connection. If the CIDR blocks overlap,
+the peering process will fail. You can always find the CIDR block of your AWS VPC
```

Comment:
Maybe screen shot of AWS VPC in console?  Or are there any direct links, AWS docs, or breadcrumbs we can point user to?

Reply:
@mfreed I have a bunch of screenshots, including a screenshot of this exact view. They are currently in a zip linked in the PR description at the very top of this PR. I don't have access to the location where they are hosted, so for now the images are not being rendered.

@ryanbooz and others are planning on helping me get those images setup so that we can properly reference them in the docs. That is a pending task though.

Reply:
just uploaded what we have ATM.